### DISCUSSION TITLE
OIDC authentication?

### DISCUSSION THREAD

**Discussion by jeffmccune** [home-assistant/architecture#865] [General] [answered]
I've got home assistant running behind caddy which is performing OIDC authentication and authorization.  Caddy passes the signed jwt as an authorization bearer token to the backend.

I've got Grafana configured to automatically authenticate and authorize based on this this token.  Is it possible or planned to support something similar with Homeassistant.

For my simple use case I'd like to configure HA to trust tokens issued by a standard OIDC issuer URL.  Ideally, map identities to an admin or a view only role based on a groups claim in the token.

This would greatly simplify user management in home assistant and make it easy to grant access to friends and family without having them create another account.

**Comment by frenck** [comment, marked as answer]
Hi!

This looks and sounds like a feature request. Please note, this is not what this repository is for.

If you want to suggest a feature, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍

../Frenck
